spinta/manifests/tabular/helpers.py

```diff
@@ -468,6 +468,11 @@ def read(self, row: Dict[str, str]) -> None:
                 f"defined for this {self.state.model.name!r} model."
             )
 
+        required = False
+        if 'required' in row['type']:
+            required = True
+            row['type'] = row['type'].replace('required', '').strip()
```

Comment:
Could you please move this:

https://github.com/atviriduomenys/spinta/blob/373b654f8369a4b218ee7badba58cc847e71045e/spinta/types/model.py#L183-L190

Out of `load` into here and parse `type` string in one place, because type can be given in following forms:

```python
"string" -> {
  "type": "string",
}
"geometry(point, 3542)" -> {
  "type": "geometry",
  "type_args": ["point", "3542"],
}
"geometry(point, 3542) unique required" -> {
  "type": "geometry",
  "type_args": ["point", "3542"],
  "unique": True,
  "required": True,
}
```

Reply:
And please collaborate on this with https://github.com/atviriduomenys/spinta/pull/343